pkg/streams/streams.go

```diff
@@ -19,6 +20,12 @@ func validateStreamName(streamName string) (string, error) {
 
 	if streamName == "" {
 		err = galasaErrors.NewGalasaError(galasaErrors.GALASA_ERROR_MISSING_STREAM_NAME_FLAG)
+	} else {
+		// Check that stream name only contains alphanumeric characters, underscores, and hyphens
+        validChars := regexp.MustCompile(`^[a-zA-Z0-9_-]+$`)
+        if !validChars.MatchString(streamName) {
+            err = galasaErrors.NewGalasaError(galasaErrors.GALASA_ERROR_INVALID_STREAM_NAME)
```

Comment:
Instead of regex, we can use the existing utils.IsNameValid() method which performs the same checks